composer.json

```
@@ -7,7 +7,7 @@
         "psr-4": {"Sil\\PhpEnv\\": "src/"}
     },
     "require": {
-        "php": "^7.4"
+        "php": "^8.1"
```

Like with some of the others, let's not block use of this library in any supported version of PHP:
```suggestion
        "php": "^7.4 || ^8.0"
```